scripts/download_image.py

```diff
@@ -63,58 +76,73 @@ def _validate_report_function(self):
                 ' total and message arguments')
 
 
-def _progress_reporter(count, total, message):
-    bar_len = 45
+def _progress_reporter(count: int, total: int, message: FutureResult,
```

Comment:
naming of `message`
message += '\r' was look like message is string

"future" is better here,
Readers search the name in source, then they will find future.result() where the value was passed from.

Reply:
Removed